src/pages/agreement/Agreement.js

```diff
@@ -0,0 +1,107 @@
+import React, { useState, useRef } from "react";
+import Select from "react-select";
+import { Form, Button } from "react-bootstrap";
+import ShowAgreement from "./ShowAgreement";
+const { data } = require("./exampleData");
+
+const universityList = [
+  { label: "UCLA", value: "UCLA" },
+  { label: "UC Berkeley", value: "UC Berkeley" },
+  { label: "UCSD", value: "UCSD" },
+];
+
+const ccList = [
+  { label: "DeAnza College", value: "DeAnza College" },
+  { label: "Foothill College", value: "Foothill College" },
+];
+
+//this can be done by ajax
+const majorList = [
+  { label: "Physics/B.A.", value: "Physics/B.A." },
+  { label: "Mathematics/B.S.", value: "Mathematics/B.S." },
+  { label: "Data Theory/B.S.", value: "Data Theory/B.S." },
+  { label: "Computer Science/B.S.", value: "Computer Science/B.S." },
+  {
+    label: "Asian American Studies/B.A.",
+    value: "Asian American Studies/B.A.",
+  },
+  { label: "Anthropology/B.S.", value: "Anthropology/B.S." },
+  { label: "Labor Studies/B.A.", value: "Labor Studies/B.A." },
+  { label: "Global Studies/B.A.", value: "Global Studies/B.A." },
+  { label: "Psychobiology/B.S.", value: "Psychobiology/B.S." },
+];
+
+function Agreement() {
+  const [onSearchTab, setOnSearchTab] = useState(true);
+  const [loadingMajor, setLoadingMajor] = useState(false);
+
+  const [university, setUniversity] = useState("");
+  const [cc, setCc] = useState("");
+  const [major, setMajor] = useState("");
+  const [agreement, setAgreement] = useState([]);
+
+  const univRef = useRef();
+  const ccRef = useRef();
+  const majorRef = useRef();
+
+  const searchHandler = (event) => {
+    console.log(university, cc, major);
+    //fetch Data from DB, suppose we have data here
```

Comment:
// Todo(Kevin): fetch Data from DB, suppose we have data here

Reply:
@biankaiwen111 Can you manually input those data into db and make adjustment to the code here? So we don't have to have this example.is file. 

After that, you can start writing APIs for inputting the data and then refine the data display method. 

The step after that should then be finding a good method/visualization for volunteer inputting